index.html

```diff
@@ -95,48 +95,43 @@ <h3>
         </pre>
         <div class="note">
           The <var>data</var> argument is marked as "optional", but it is
-          effectively required; <a for="Navigator"><code>share</code></a> will
-          throw a <a data-cite=
+          effectively required; <a>share()</a> will throw a <a data-cite=
           "!WEBIDL#exceptiondef-typeerror"><code>TypeError</code></a> if called
           with no arguments. WebIDL <a data-cite=
           "!WEBIDL#idl-operations">mandates that it be optional</a> because the
           dictionary members are all optional. See WebIDL <a href=
           "https://github.com/heycam/webidl/issues/130">Issue #130</a>.
         </div>
         <p>
-          User agents that do not support sharing SHOULD NOT expose <a for=
-          "Navigator"><code>share</code></a> on the
-          <a><code>Navigator</code></a> interface.
+          User agents that do not support sharing SHOULD NOT expose
+          <a>share()</a> on the <a>Navigator</a> interface.
         </p>
         <div class="note">
           The above statement is designed to permit feature detection. If
-          <a for="Navigator"><code>share</code></a> is present, there is a
-          reasonable expectation that it will work and present the user with at
-          least one <a>share target</a>. Clients can use the presence or
-          absence of this method to determine whether to show UI that triggers
-          its use.
+          <a>share()</a> is present, there is a reasonable expectation that it
+          will work and present the user with at least one <a>share target</a>.
+          Clients can use the presence or absence of this method to determine
+          whether to show UI that triggers its use.
         </div>
         <section>
           <h4>
-            <code>share</code> method
+            <dfn>share()</dfn> method
           </h4>
           <p>
-            When the <dfn>share</dfn> method is called with argument
-            <var>data</var>, run the following steps:
+            When the <a>share()</a> method is called with argument
+            <a>ShareData</a> <var>data</var>, run the following steps:
```

Comment:
I don't think this needs to say `ShareData` here (it seems very unconventional to specify a data type when defining an argument to an algorithm).

Reply:
Np. I'll remove that. I've seen it as a trend I think in the WHATWG specs.